### PR TITLE
Input component

### PR DESCRIPTION
**Pull request by aamir74**
normal, error and success style types of inputs

### CODE REVIEW

**Review by SaiArunesh — APPROVED** [review]

File: components/input.css [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+.text-label {
+  margin: 1rem;
+  margin-bottom: 0;
+  font-size: 1rem;
+  line-height: 1;
+  font-weight: 400;
+}
+
+.textbox {
+  padding: 0.5rem 0.5rem;
```
Comment (VishalPatil18) [inline components/input.css]: You can also do `padding: 0.5rem;` here as all four sides have same padding

**Review by VishalPatil18 — APPROVED** [review]
Just added one suggestion. Apart from that everything looks good. Great Job @aamir74 👍

**Review by ChhikaraBRUH — APPROVED** [review]
code is fine

File: components/input.css (hunk 1 above)
Comment (aamir74) [reply components/input.css]: yes.

**Pull request merged**